### ISSUE TITLE
Update React + React Script + Typescript + Mui

### ISSUE THREAD

**Issue by rulyotano** [rulyotano/minesweeper#32]

**Comment by DonMiller9294** [comment]
I would like to help you out, could you give description of issue.

**Comment by rulyotano** [comment]
> I would like to help you out, could you give description of issue.

Hi @DonMiller9294 ! You are more than welcome to collaborate! 💪The main issue is that currently, the front project is using an old version of the react-create-app library, which uses an old version of react. Also, Material-Ui library is not updated. Currently v4, when latest one is v5. When I tried to update Mui or React I got tons of errors. My idea in this case is to create a new create-react-app from scratch and move things there, but using MUI v5 instead. Should be simple, but ...

**Comment by DonMiller9294** [comment]
thank you for the update i will take a look at it sometime this evening.

**Comment by DonMiller9294** [comment]
@rulyotano you could upgrade the version of react you are using. have you checkout out the official react documentation about this?

**Comment by rulyotano** [comment]
Yes, that is what we want to achieve. Upgrade React and MUI versions. But there are a lot of incompatibilities (typescript and so on...). That is why I think it is better to start a new project from scratch and move things there.
